test/e2e/framework/metrics/metrics_proxy.go

```diff
@@ -34,37 +34,47 @@ import (
 )
 
 type componentInfo struct {
+	Name string
```

Comment:
this doesn't seem to be used later

Reply:
line 70 will log the components' name, but if you like, I could remove this field, since we could tell which component it is from the port number.

Reply:
no no no, I miss that sorry, I thought that wasn't used at all